ci/benchmark/build.sh

```diff
@@ -92,8 +92,7 @@ pip install "git+https://github.com/python-streamz/streamz.git@master" --upgrade
 
 logger "Check versions..."
 python --version
-$CC --version
-$CXX --version
+
```

Comment:
Is there a reason these are dropped?

Note: This happens in other files below, but am just commenting on this one for simplicity

Reply:
Had to remove these because since we switched to `conda` compilers, the custom compilers have been removed here: https://github.com/rapidsai/gpuci-build-environment/pull/265, and the images seem to have shipped because of which I was seeing errors.